sandbox/apparmor/apparmor_test.go

```diff
@@ -303,30 +310,33 @@ exit "$EXIT_CODE"
 		c.Check(mockParserCmd.Calls(), DeepEquals, expectedCalls)
 		data, err := ioutil.ReadFile(filepath.Join(d, "stdin"))
 		c.Assert(err, IsNil)
-		c.Check(string(data), Equals, `profile snap-test {
+		c.Check(string(data), Equals, `profile snap-test  {
```

Comment:
Then all of these could stay as they were

Reply:
Great suggestion - updated in https://github.com/snapcore/snapd/pull/13333/commits/fbbf3094043271f86108f9802b4965338c9098ee